### PR TITLE
Debug path

### PR DESCRIPTION
**Pull request by tristanmorgan**
### Description

Adding a /debug path for use in the Tech Challenge. it outputs some of the config and the local **hostname** and **public IP** address.

### Checklist

- [x] Code compiles correctly
- [x] Created tests which fail without the change (if possible)
- [x] All tests passing
- [x] Extended the README / documentation, if necessary
- [x] Added issue number to `Fixes` line above

### CODE REVIEW

File: ui/ui.go [hunk 1]
```diff
@@ -81,6 +85,8 @@ func debugHandler(cfg Config) http.Handler {
 
 		cfg.DB.DbUser = "********"
 		cfg.DB.DbPassword = "********"
+		cfg.Publicip = publicIP()
+		cfg.Hostname, _ = os.Hostname()
```
Comment (andrewpym-servian) [inline ui/ui.go:89]: Could we add the db connection string. without password?  plus the data and or time?

Comment (tristanmorgan) [reply ui/ui.go:89]: The rest of the structure contains the database details.

I'll add a time string in there though.

Comment (tristanmorgan) [reply ui/ui.go:90]: https://github.com/servian/TechChallengeInstructions

File: ui/ui.go [hunk 2]
```diff
@@ -72,3 +80,35 @@ func healthcheckHandler(cfg Config) http.Handler {
...
+		}
+		fmt.Fprintf(w, string(bytes))
+	})
+}
+
+func publicIP() string {
+	url := "https://checkip.amazonaws.com/"
+	resp, err := http.Get(url)
+	if err != nil {
+		panic(err)
```
Comment (servian-sheldon-gill) [inline ui/ui.go:106]: Why panic() instead of returning a string like "unavailable" or "unknown" ?

**Review by servian-sheldon-gill — APPROVED** [review]

**Pull request merged**